content/docs/concurrent-mode-adoption.md

```diff
@@ -15,98 +15,97 @@ next: concurrent-mode-reference.html
+* **Eşzamanlı mod:** `ReactDOM.createRoot(rootNode).render(<App />)`. 
+Bu, şu anda deneysel. Gelecekte, kararlılığa ulaştıktan sonra onu öntanımlı React modu yapmayı istiyoruz. Bu, yeni özelliklerin *tamamını* etkinleştiriyor.
 
-### Why So Many Modes? {#why-so-many-modes}
+### Neden Bu Kadar Çok Mod Var? {#why-so-many-modes}
 
-We think it is better to offer a [gradual migration strategy](/docs/faq-versioning.html#commitment-to-stability) than to make huge breaking changes — or to let React stagnate into irrelevance.
+Biz, çok büyük ve bozucu değişiklikler yapmak yerine [kademeli migrasyon stratejisi](/docs/faq-versioning.html#commitment-to-stability) sunmanın - veya React'in gereksizliğe doğru durulmasının - daha iyi olduğunu düşünüyoruz.
 
-In practice, we expect that most apps using Legacy Mode today should be able to migrate at least to the Blocking Mode (if not Concurrent Mode). This fragmentation can be annoying for libraries that aim to support all Modes in the short term. However, gradually moving the ecosystem away from the Legacy Mode will also *solve* problems that affect major libraries in the React ecosystem, such as [confusing Suspense behavior when reading layout](https://github.com/facebook/react/issues/14536) and [lack of consistent batching guarantees](https://github.com/facebook/react/issues/15080). There's a number of bugs that can't be fixed in Legacy Mode without changing semantics, but don't exist in Blocking and Concurrent Modes.
+Pratikte miras modunu kullanan uygulamaların çoğunun en azından engelleme moduna (hatta eşzamanlı moda) migrasyonu mümkün olmalı. Bu parçalanma, tüm modları desteklemeyi hedefleyen kütüphaneler için kısa vadede can sıkıcı olabilir. Ancak, ekosistemi miras modundan kademeli olarak uzaklaşmak aynı zamanda React ekosistemindeki büyük kütüphaneleri etkileyen [layoutu okurken kafa karıştıran Suspense davranışı](https://github.com/facebook/react/issues/14536) ve [tutarlı harmanlama garantisinin olmayışı](https://github.com/facebook/react/issues/15080) gibi sorunları da *çözecektir*. Miras modunda bulunan kimi hatalar mantıksal değişiklikler yapılmadan çözülemiyor ama engelleme modunda ve eşzamanlı modda bulunmuyor. 
 
-You can think of the Blocking Mode as a "gracefully degraded" version of the Concurrent Mode. **As a result, in longer term we should be able to converge and stop thinking about different Modes altogether.** But for now, Modes are an important migration strategy. They let everyone decide when a migration is worth it, and upgrade at their own pace.
+Engelleme modunu, eşzamanlı modun "zarifçe indirgenmiş" bir versiyonu olarak düşünebilirsiniz. **Sonuç olarak, uzun vadede birleştirebileceğiz ve farklı modları düşünmeyi komple bırakabileceğız.**  Ama şimdilik modlar önemli bir migrasyon stratejisi. Migrasyonun değip değmeyeceğine herkesin kendinin karar vermesine ve kendi hızlarıyla yükseltmelerine izin veriyorlar.
```

Comment:
Same here. It could be:

`Migrasyonun değip değmeyeceğine` -> `Geçiş yapmaya değip değmeyeceğine ....`